src/plugins/provider.js

```diff
@@ -42,11 +42,16 @@ function createVueHooksMap (hookCallback) {
 
 function ngVueProvider ($injector) {
   let inQuirkMode = false
+  let vuexStore = null
```

Comment:
I thin this plugin provider should be rewritten because it's not extensible at all.
If we wanna add some other core features, we should write a new plugin instead of rewriting the plugin provider.